src/frontend/src/stream_fragmenter/mod.rs

```diff
@@ -290,14 +290,21 @@ fn build_fragment(
             current_fragment.upstream_table_ids.push(node.table_id);
         }
 
-        NodeBody::StreamCdcScan(node) => {
+        NodeBody::StreamCdcScan(_) => {
             current_fragment.fragment_type_mask |= FragmentTypeFlag::StreamScan as u32;
```

Comment:
Looks like something was missed in the refactoring. I suppose `StreamScan` should also be refactored to `FragmentTypeFlag::StreamCdcScan`? Out of scope for this PR.